src/domains/Work/components/WorkEdit/presentations/items/AssetUpload.tsx

```diff
@@ -23,27 +24,10 @@ export const AssetUpload: FC<Props> = ({ handleUploadAssets, assets }) => (
       <Typography className="text-red-500" variant="body2">
         必須
       </Typography>
+      <InfoPopOver>
+        {`対応形式:\n    画像[.png .jpg .jpeg .bmp .gif]\n    動画[.mp4 .mov]\n    音源[.mp3 .wav .m4a ]\n    モデル[.gltf .fbx]\n    zip[.zip]`}
```

Comment:
`<br />`じゃだめ？(特に要件とか仕様は確認してないので，何か理由があるなら無視してOK．)
```suggestion
        対応形式:
        <br />
        画像[.png .jpg .jpeg .bmp .gif]
        <br />
        動画[.mp4 .mov]
        <br />
        音源[.mp3 .wav .m4a ]
        <br />
        モデル[.gltf .fbx]
        <br />
        zip[.zip]
```